test/tensor_util.hpp

```diff
@@ -71,4 +71,114 @@ void operate_over_subtensor_impl(const data_operator_t<T>& r_data_operator,
     }
 }
 
+template <typename T>
+void output_tensor_to_csv(const tensor<T>& x, std::string filename)
```

Comment:
added these file for now since these functions were deleted and not replaced by PR https://github.com/ROCm/MIOpen/pull/2896#discussion_r1575539336

Reply:
Why do we need these functions if they are not used anywhere?

Reply:
I guess they can be used during manual debugging. Like you want to print some tensor, but you don't want to keep all the intermediate code like hacky printing from the guts of solver in the release code.